.circleci/config.yml

```diff
@@ -0,0 +1,189 @@
+version: 2.1
+
+orbs:
+  aws-s3: circleci/aws-s3@1.0.11
+
+only_branches: &only_branches
+  filters:
+    branches:
+      ignore: master
+
+only_master: &only_master
+  filters:
+    branches:
+      only: master
+
+only_v_tags: &only_v_tags
+  filters:
+    tags:
+      only: /^v.*/
+    branches:
+      ignore: /.*/
+
+jobs:
+  build-ltops-and-test:
+    docker:
+      - image: circleci/golang:1.12
+    steps:
+      - checkout
+      - restore_cache:
+          key: dependencies-{{ checksum "go.sum" }}
+      - run:
+          name: Install dependencies
+          command: |
+            export PATH=$PATH:$GOPATH/bin
+            mkdir -p $GOPATH/bin
+            make install
+      - save_cache:
+          key: dependencies-{{ checksum "go.sum" }}
+          paths:
+            - $GOPATH/bin
```

Comment:
I think the path should be `- "/go/pkg/mod"`  the `bin` is the binaries only

Reply:
thank you, please check if it is correct now.